### PR TITLE
[NG] Migrated UX Engineer page to layer5-ng 

### LINKED ISSUE #861: [NG] Migrate the UX Engineer page to layer5-ng
**Current Behavior**
The current `layer5-ng` site contains a single post on the [careers page ](https://layer5ng.netlify.app/careers)

**Desired Behavior**
Add the[  UX Engineer page ](https://layer5.io/careers/ux-engineer) from the [current site](https://layer5.io/careers) into layer5-ng. 

The data for the various job post are found inside [this folder](https://github.com/layer5io/layer5/tree/master/collections/_careers). Reference on how to a post of in the careers section in layer5-ng can be taken from an already added post of Digital Marketing Internship in [this folder](https://github.com/layer5io/layer5/tree/layer5-ng/src/collections/careers). (Change `type: internship` to `type: job`) 

---
[Optional] **Alternatives**

[Optional] **Additional context**

---
**Contributor Resources**
The `layer5` repo contains two websites. The current generation and the next-generation of the layer5.io site.

If the `layer5-ng` label is absent on this issue, then this issue pertains to the current generation of the layer5.io website, which uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [`layer5` contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

If the `layer5-ng` label is present on this issue, then this issue pertains to the next-generation of the layer5.io website, which uses Gatsby, Strapi, and GitHub Pages. Site content is found under the [`layer5-ng` branch](https://github.com/layer5io/layer5/tree/layer5-ng).
- See [`layer5-ng` contributing instructions](https://github.com/layer5io/layer5/blob/layer5-ng/CONTRIBUTING.md)

### PR DESCRIPTION
**Pull request by thescripted**
**Description**

This PR fixes #861 

**Notes for Reviewers**
Let me know if anything is missed!

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: src/collections/careers/ux-engineer/index.mdx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+
+<CareerWrapper>
+
+<h3>About Layer5</h3>
+We are a fun, talented and driven team dedicated to providing our customers with easy-to-use software and a world-class, open source community experience. While small, the Layer5 community represents the largest collection of service mesh projects and their maintainers in the world. Advanced and emerging technology, and empowering engineers are at the core of our company, and though we move at a very fast pace, we are committed to maintaining a work environment of continual learning and giving back to our community. A passion for great design, dedication to innovation and a strong social media presence are our building blocks for creating a unique and engaging brand across the cloud native landscape. Our headquarters are located in the hill country of Austin, Texas, and we embrace the unique vibe and energy of our hometown as an inspiration for our brand and company culture.
+
+<div className="logo">
+    <img src={ux_designer_logo} alt="ux-designer-logo" />
+    Digital Marketing Internship
```
Comment (Tanuj22) [inline src/collections/careers/ux-engineer/index.mdx:21]: Should be `UX Engineer`

File: src/collections/careers/ux-engineer/index.mdx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+innovation is fosterd and hard work is encouraged. We do so in ways that align with your individual interests and career goals.
+User experience is at the forefront of how we create intuitive, innovative, and beautiful products that people love.</p>
+
+<p>As a UX Engineer, you'll work on the front end of our products and features used by people everyday. 
+The work is challenging, fast-paced, and always changing. You have demonstrated analytical skills, 
+an unwavering commitment to quality, a collaborative work ethic, and cutting-edge coding skills. 
+You live and breathe the fundamentals of JavaScript (not just jQuery), CSS, HTML, and command a portfolio of 
+web technologies new and old - all in order to launch innovative features to a global audience.</p>
+
+<h5 class="black-text">Responsibilities</h5>
```
Comment (Tanuj22) [inline src/collections/careers/ux-engineer/index.mdx:37]: Remove this class

File: src/collections/careers/ux-engineer/index.mdx [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    <li>Work closely with UX designers, project managers, and back-end engineers to implement versatile front-end solutions to web development issues.</li>
+    <li>Embrace emerging standards while promoting best practices in order to push the limits of what a browser can do.</li>
+    <li>Manage your own time, and work well both independently and as part of a team.</li>
+    <li>Launch, iterate, and make a difference. Provide front-end coding expertise, and help make our products better for users.</li>
+    <li>Design and develop the front ends of user-facing products.</li>
+    <li>Interns will enhance their human-centered design (HCD) techniques, producing software so intuitive that documentation isn't necessary. </li>
+    <li>You will engage with users and the community to solicit and incorporate feedback on designs in an iterative and evolving process.</li>
+</ul>
+
+<h5 class="black-text">Minimum Qualifications</h5>
```
Comment (Tanuj22) [inline src/collections/careers/ux-engineer/index.mdx:50]: Remove this class

File: src/collections/careers/ux-engineer/index.mdx [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+<ul>
+    <li>Bachelor's degree in Computer Science, HCI, Design, or equivalent practical experience.</li>
+    <li>Experience with JavaScript, HTML, and CSS.</li>
+    <li>Experience in designing and implementing user interfaces (UIs).</li>
+    <li>Experience in development or prototyping.</li>
+</ul>
+
+<h5 class="black-text">Desired Qualifications</h5>
```
Comment (Tanuj22) [inline src/collections/careers/ux-engineer/index.mdx:58]: Remove this class

**Review by Tanuj22 — CHANGES_REQUESTED** [review]
Great work @thescripted . Some minor changes and we are good to go.

**Comment by thescripted** [comment]
Thanks for the heads up, I believe I got those cleared up now.

**Review by Tanuj22 — APPROVED** [review]
Thank You :)

**Comment by Tanuj22** [comment]
Woohoo @thescripted.  If you haven't already, join the Layer5 community http://slack.layer5.io if you would. It's a welcoming one :)

**Pull request merged**